src/js/visual/configurator-prc.js

```diff
@@ -173,6 +173,8 @@ ripe.ConfiguratorPrc.prototype.updateOptions = async function(options, update =
         options.configAnimate === undefined ? this.configAnimate : options.configAnimate;
     this.viewAnimate = options.viewAnimate === undefined ? this.viewAnimate : options.viewAnimate;
 
+    this.useMasks = this.owner.brand && this.owner.hasTag("no_masks") ? false : this.useMasks;

```

Comment:
Why is the `brand` thing needed?

Reply:
To see if the SDK is already configured, or else it will send out errors since `loadedConfig` is null

Reply:
Shouldn't you be look at the `loadedConfig` then? (or isReady)